.circleci/config.yml

```diff
@@ -333,6 +333,7 @@ jobs:
           name: Publish to Cloudsmith
           command: |
             ./gradlew --no-daemon --parallel cloudsmithUpload publish
+      - notify
```

Comment:
hmm, this would have to be on every job so we get notified if master fails because of something like an intermittent unit test etc.  Most of the time it's Cloudsmith that's being difficult but would be good to be told about any failures.

Reply:
notify itself limits to just being on master, i think it probably resolves ok...

Reply:
I mean we should add this `- notify` step to every job so that when any of them fail on master we get a notification. Unless I'm misunderstanding something.

Reply:
also the publish tasks don't get run on other branches...